index.html

```diff
@@ -196,5 +196,47 @@ <h2>About me</h2>
             </ul>
         </div>
     </section>
+
+    <footer>
+        <div class="icons">
+        <p class="attract">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            Full name
+            Email address
+            Enter text here
+            Get in touch
+        </p>
+        <form action="https://formspree.io/f/xeqdogov" method="post">
+                <div>
+                    <input type="text" name="fullname" placeholder="Full name" maxlength="30" required class="formel">
+                </div>
+                <div>
+                    <input type="email" name="email" placeholder="Email address" required class="formel">
+                </div>
+                <div>
+                   
+                
+                <textarea  name="message" maxlength="500" class="textarea" placeholder="Enter your text" required>
+                </textarea>
+                </div>
+
+                <button type="submit">
+                      Get in touch
+                </button>
+        </form>
+        <hr class="line">
+           </div>
+
+           <div class="icons">
+            <ul>
+                <li><img src="./Images/githubblack.png" alt="soclai media icons"></li>
+        <li><img src="./Images/linkblack.png" alt="soclai media icons"></li>
+        <li><img src="./Images/handblack.png" alt="soclai media icons"></li>
+        <li><img src="./Images/mediumblack.png" alt="soclai media icons"></li>
+        <li><img src="./Images/twitterblack.png" alt="soclai media icons"></li>
```

Comment:
- [ ] Please make each of your icon inside an `a` tag since these images are links to your social media accounts :+1: